scripts/pages.js

```diff
@@ -6,7 +6,7 @@ const recipientRequestObject = {
     updatedFrequency: 'daily',
     priority: '0.8',
     isAsync: true,
-    url: 'https://api.usaspending.gov/api/v2/recipient/duns/',
+    url: 'https://api.usaspending.gov:443/api/v2/recipient/duns/',
```

Comment:
This is t he only place we need to include the port number -- in these static objects w/ the property `url`